yarn-project/end-to-end/src/e2e_fees/gas_estimation.test.ts

```diff
@@ -53,12 +53,10 @@ describe('e2e_fees gas_estimation', () => {
   const makeTransferRequest = () => bananaCoin.methods.transfer_in_public(aliceAddress, bobAddress, 1n, 0n);
 
   // Sends two tx with transfers of public tokens: one with estimateGas on, one with estimateGas off
-  const sendTransfers = (paymentMethod: FeePaymentMethod) =>
+  const sendTransfers = (paymentMethod: FeePaymentMethod, estimatedGasPadding: number) =>
```

Comment:
In this test I've probably uncovered a gas estimator issue. Issue tracked [here](https://github.com/AztecProtocol/aztec-packages/issues/11324).

Will revisit the issue once the `Packable` trait is implemented.